config/spec-bundle.js

```diff
@@ -17,7 +17,7 @@ Error.stackTraceLimit = Infinity;
 // https://github.com/angular/angular-cli/issues/1675
 require('intl');
 require('intl/locale-data/jsonp/en');
-
+require('pact-web');
```

Comment:
Short comment why or link to the PR emphasizing why the order is important could help?